ui/src/views/QueueView.vue

```diff
@@ -108,7 +108,26 @@ export default {
     retrieveQueryTree() {
       this.getQueryTree(
         (success) => {
-          this.queryTree = success.body.query_tree;
+          console.log("success.body.query_tree", success.body.query_tree); ////
```

Comment:
we can remove all `////` lines?

Reply:
Removed. I left only some debug prints I think are still useful, we can delete them later